site/content/contribute/server/feature-flags.md

```diff
@@ -26,13 +26,115 @@ Feature flags allow us to be more confident in shipping features continuously to
+
+### How to log in?
+
+Login is controlled via OneLogin. If you have access you will be able to log in through there.
+### Terminology
+
+In Split, there's some terminology you will need to understand:
+
+ - **Split** - A feature flag
+ - **Treatment** - The values a split (e.g. feature flag) can have
+ - **Environment** - Different environments that splits can target such as test, staging and production
+ - **Server** - Single Mattermost workspace
+ - **Segment** - A list of servers that can be targeted by a split
+
+### Adding a split
+
+After you've added the feature flag in code, you'll need to create a split for it.
+
+ 1. Log in to Split.
+ 2. On left sidebar, click "Splits".
+ 3. At the top click "Create split".
+ 4. Fill in the following:
+   1. Enter the name of your feature flag. It must match the name used in code in `model/feature_flags.go`.
+   2. Select "server" as the traffic type.
+   3. Optionally add tags and a description.
+   4. Click "Create".
+
+The split is now created but not targeting anything. The next section covers adding basic targeting rules.
+
+###  Add targeting rules to a split
+
+To have the split have any effect, it needs targeting rules to know which environment and servers it should set the feature flag value to. To add basic targeting rules:
+
+ 1. Find your split and click on it.
+ 2. At the top, under the name of the split select the "Environment" you want to target.
+   - See [Environments](#Environments) for a description of each.
+ 3. In the center of the screen, select "Add Rules".
+ 4. Define your treatments:
+   - Recall that treatments are just the values your feature flag can have.
+   - Boolean flags should use the default "on" and "off" treatments.
+   - For non-boolean flags, change the treatments to be the possible values. E.g. for a split that specifies a plugin's version you could have three treatments of "1.0", "1.1" and "2.0".
+ 5. Down the page a bit, set the default rule to one of your treatments.
+   - This will set the flag for all servers in the environment while you don't have any other targeting rules configured.
+ 6. At the top right, click "Save changes" and confirm them on the next screen.
+
+That covers targeting an environment with a split and adding a default rule for all servers in that environment. The next section covers targeting specific servers.
+
+### Target a specific server
+
+The ID for a "server" as seen by Split is the Mattermost server's telemetry ID. To target a specific server you need that telemetry ID. The main way to get the telemetry ID from a server would be from the analytics but you can also manually grab the telemetry ID by one of the following methods:
```

Comment:
"...would be from the analytics..." - Analytics team?

Reply:
Not exactly sure how or where it's exposed, maybe it's a new dashboard or search in Looker. @enelson720 or @jasonblais any idea for how we'd be able to give people the ability to find the telemetry ID from the analytics data?

Reply:
Telemetry ID is the same as a server ID. @enelson720 will defer to you on the exact steps to retrieve from Looker.

Reply:
Yeah, it's fairly straight forward @jwilander. I can provide a look that can be filtered by customer name and/or workspace name in the case of cloud workspaces. [Here's a jumping off point for retrieving the telemetry id.](https://mattermost.looker.com/explore/data_warehouse/server_fact?toggle=fil&qid=m250frAeAMARzhZBek5C4e)